From 7aa5d8a60374f4ed8ee1da1ef6e431a93754f190 Mon Sep 17 00:00:00 2001
From: Peter Antoine <peter.antoine@intel.com>
Date: Sun, 21 Feb 2016 12:57:59 +0000
Subject: [PATCH 090/143] [VPG]: drm/i915: Android MOCS Tables for CL519294

This change adds the MOCS entries required by MESA to the MOCS tables
and requires the corresponding GMM change to include both MESA and UFO
entries in the MOCS table.

Signed-off-by: Peter Antoine <peter.antoine@intel.com>
Change-Id: I0ec2102d7090f94f04ce1ca8a3c1ce5abb8db130
---
 drivers/gpu/drm/i915/intel_mocs_gmm_table.c |    7 +++----
 1 files changed, 3 insertions(+), 4 deletions(-)

Index: current/drivers/gpu/drm/i915/intel_mocs_gmm_table.c
===================================================================
--- current.orig/drivers/gpu/drm/i915/intel_mocs_gmm_table.c
+++ current/drivers/gpu/drm/i915/intel_mocs_gmm_table.c
@@ -28,13 +28,12 @@ static const struct drm_i915_mocs_entry
 	{0x00000009, 0x0010}, /* ED_UC LLC/eLLC EDSCC:0 L3SCC:0 L3_UC */
 	{0x00000038, 0x0030}, /* ED_PTE LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
 	{0x0000003b, 0x0030}, /* ED_WB LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
+	{0x00000037, 0x0030}, /* ED_WB LLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
 	{0x00000039, 0x0030}, /* ED_UC LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
-	{0x0000003b, 0x0010}, /* ED_WB LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
-	{0x00000039, 0x0010}, /* ED_UC LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
-	{0x0000001b, 0x0030}, /* ED_WB LLC/eLLC LRU_S EDSCC:0 L3SCC:0 L3_WB */
 	{0x00000037, 0x0010}, /* ED_WB LLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
-	{0x00000037, 0x0030}, /* ED_WB LLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
+	{0x00000039, 0x0010}, /* ED_UC LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
 	{0x00000017, 0x0010}, /* ED_WB LLC LRU_S EDSCC:0 L3SCC:0 L3_UC */
+	{0x0000003b, 0x0010}, /* ED_WB LLC/eLLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
 	{0x00000033, 0x0030}, /* ED_WB eLLC LRU_L EDSCC:0 L3SCC:0 L3_WB */
 	{0x00000033, 0x0010}, /* ED_WB eLLC LRU_L EDSCC:0 L3SCC:0 L3_UC */
 	{0x00000017, 0x0030}, /* ED_WB LLC LRU_S EDSCC:0 L3SCC:0 L3_WB */
